wlauto/workloads/googlephotos/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -413,4 +416,17 @@ private void saveAndReturn() throws Exception {
         navigateUpButton.waitForExists(viewTimeout);
         navigateUpButton.click();
     }
+
+    // Helper to tag an individual photograph based on index in wa-working gallery.
```

Comment:
If you add that description then I won't be bothered by the actual name.

From: johnrchrdsn <notifications@github.com<mailto:notifications@github.com>>
Reply-To: jimboatarm/workload-automation <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, 19 May 2016 at 13:19
To: jimboatarm/workload-automation <workload-automation@noreply.github.com<mailto:workload-automation@noreply.github.com>>
Cc: user <james.hartley@arm.com<mailto:james.hartley@arm.com>>, Assign <assign@noreply.github.com<mailto:assign@noreply.github.com>>
Subject: Re: [jimboatarm/workload-automation] Add Multiapp workload (#23)


In wlauto/workloads/googlephotos/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java<https://github.com/jimboatarm/workload-automation/pull/23#discussion_r63867747>:

> @@ -413,4 +416,17 @@ private void saveAndReturn() throws Exception {
>          navigateUpButton.waitForExists(viewTimeout);
>          navigateUpButton.click();
>      }
> +
> +    // Helper to tag an individual photograph based on index in wa-working gallery.


Maybe I can change this to 'mark' or 'check'. After long clicking it should mark the photo with a little tick in the corner of the image to demonstrate it has been selected. I can add a fuller comment also.

-
You are receiving this because you were assigned.
Reply to this email directly or view it on GitHub<https://github.com/jimboatarm/workload-automation/pull/23/files/2532f36a6219d594a0af6876fef9832f26e992d9#r63867747>

IMPORTANT NOTICE: The contents of this email and any attachments are confidential and may also be privileged. If you are not the intended recipient, please notify the sender immediately and do not disclose the contents to any other person, use it for any purpose, or store or copy the information in any medium. Thank you.